common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -83,7 +83,7 @@ def __init__(
         """
         self.database = MongoProxy(
             pymongo.database.Database(
-                pymongo.MongoClient(
+                pymongo.MongoReplicaSetClient(
```

Comment:
My thought was that the `DOC_STORE_CONFIG` is shared across the various modulestores, so the connection setup should be as well.